templates/reporting/user_list_period_data.html.twig

```diff
@@ -81,7 +81,7 @@
             </td>
             {% for period in attribute(userPeriod, period_attribute) %}
                 <td class="text-nowrap text-center day-total{% block period_cell_class %}{% endblock %}"{% block period_cell_attribute %}{% endblock %}>
-                    {% if period.totalDuration > 0 or period.totalRate > 0 or period.totalInternalRate > 0 %}
+                    {% if period.totalDuration != 0 or period.totalRate > 0 or period.totalInternalRate > 0 %}
```

Comment:
Compare to the other template, all checks are for `!= 0` instead of `> 0`:
https://github.com/kimai/kimai/blob/main/templates/reporting/report_by_user_data.html.twig#L75

Reply:
Fixed!